i18n/en.json

```
@@ -5903,6 +5903,14 @@
     "id": "app.post.update.app_error",
     "translation": "Unable to update the Post."
   },
+  {
+    "id": "app.post_reminder.app_error",
+    "translation": " "
+  },
+  {
+    "id": "app.post_reminder_dm",
+    "translation": "Hi there, you asked me to remind you about {{.SiteURL}}/{{.TeamName}}/pl/{{.PostId}}) by @{{.Username}}"
```

There's an extra parentheses after `{{.PostId}}`

```suggestion
    "translation": "Hi there, you asked me to remind you about {{.SiteURL}}/{{.TeamName}}/pl/{{.PostId}} by @{{.Username}}"
```